kcbq-connector/src/integration-test/java/com/wepay/kafka/connect/bigquery/it/BigQueryConnectorIntegrationTest.java

```diff
@@ -280,6 +280,19 @@ public void testLogicalTypes() {
     testRows(expectedRows, readAllRows("kcbq_test_logical_types"));
   }
 
+  @Test
+  public void testDoubleInf() {
+    List<List<Object>> expectedRows = new ArrayList<>();
+
+    // {"row":1,"value":{"double":3.14}}
+    expectedRows.add(Arrays.asList(1L, 3.14));
```

Comment:
Since it is currently impossible to test those special values without that code (and we don't want that code in because it would do unintended things, I was thinking of just removing the integration test.